conductor/submitter_nuke.py

```diff
@@ -236,6 +249,12 @@ def runPreSubmission(self):
         again (after validation succeeds), we also pass the depenencies along
         in the returned dictionary (so that we don't need to collect them again).
         '''
+        # Check if script has unsaved changes and ask user if they'd like to
+        # save their script before continuing with submission
+        if not self.checkSaveBeforeSubmission():
+            raise exceptions.UserCanceledError()
+        else:
```

Comment:
No need for an `else` here if the body of the sibling `if` raises an exception.

Reply:
agreed.  Ravi was simply copying the code from the maya submitter (which has the same issue)

Reply:
Removed